addons/web/static/src/core/py_js/py_builtin.js

```diff
@@ -34,7 +34,10 @@ export const BUILTINS = {
     },
 
     context_today() {
-        return PyDate.today();
+        const d = new Date();
+        // include getTimezoneOffset so tests can set timezone by patch getTimezoneOffset
+        const t = PyDateTime.now().add(PyTimeDelta.create({minutes: -d.getTimezoneOffset()}));
```

Comment:
getTimezoneOffset is a method of the native JS Date class.
However this does not take into account the timezone in the user settings but instead the system's timezone.
While this may work for the tests this is not a good solution.